ftw/slacker/interfaces.py

```diff
@@ -0,0 +1,21 @@
+from zope.interface import Interface
+
+
+class ISlackNotifier(Interface):
+    """An ISlackNotifier utility is able to post messages through webhooks
+    into a slack channel.
+    """
+
+    def notify(webhook_url, timeout, verify, **payload):
+        """This method performs the slack-notification.
+
+        @param webhook_url: The Slack webhook url
+        @param timeout: Raises a timeout error after the given seconds.
```

Comment:
What kind of notation is that?
I have started using the [sphinx autodoc syntax](http://www.sphinx-doc.org/en/stable/ext/autodoc.html), for example in [ftw.publisher](https://github.com/4teamwork/ftw.testbrowser/blob/master/ftw/testbrowser/core.py#L210).
If this syntax has no deeper meaning I propose to also use the autodoc syntax, mainly for being a good example and good practice. I expect that we will start using that in other places too and it would be grat to have a consistent syntax 😉